test/e2e/specs/all-features-confirmation/recordset/presentation.conf.js

```diff
@@ -1,15 +1,38 @@
 var pConfig = require('./../../../utils/protractor.configuration.js');
 
 var config = pConfig.getConfig({
-    configFileName: 'recordset/dev.json',
-    specs: [
-        "presentation.spec.js"
-    ],
-    setBaseUrl: function(browser, data) {
-        browser.params.url = process.env.CHAISE_BASE_URL;
-        return browser.params.url;
+  configFileName: "recordset/dev.json",
+  specs: ["presentation.spec.js"],
+  setBaseUrl: function (browser, data) {
+    browser.params.url = process.env.CHAISE_BASE_URL;
+    return browser.params.url;
+  },
+  chaiseConfigFilePath:
+    "test/e2e/specs/all-features-confirmation/chaise-config.js",
+
+    // Below config is used to test download feature without having to show prompt 
+    // to select the local PC download location.
+  capabilities: {
```

Comment:
Why does this need to be defined? I can't get the export test cases to pass without commenting this capabilities object out